apps/governance/src/routes/proposals/components/vote-breakdown/vote-breakdown.tsx

```diff
@@ -248,23 +251,45 @@ const VoteBreakdownBatch = ({
     proposal.state === ProposalState.STATE_DECLINED ||
     proposal.state === ProposalState.STATE_PASSED
   ) {
+    let description = t(
+      'Proposal passed: conditions met for {{count}} of {{total}} proposals',
+      {
+        count: passingCount['true'] || 0,
+        total: voteInfo.length,
+      }
+    );
+    let descriptionIconColor = 'text-vega-green';
+    if (proposal.__typename === 'BatchProposal') {
+      const subStates = compact(
```

Comment:
non blocking:  worth de-duping with a helper something like: `isBatchProposalFullyPassed`. I don't love this function name though